tests/cicd/jenkins_transfer_learning_basic_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# init conda env
+eval "$('/opt/intel/oneapi/intelpython/latest/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
+# activate conda
+conda activate pytorch-1.10.0
```

Comment:
If ModelAdapter runs with pytorch1.12.0 env, this env name need to be changed.

Reply:
Updated